packages/firebase_auth/ios/Classes/FirebaseAuthPlugin.m

```diff
@@ -227,6 +237,10 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
             completion:^(NSError *error) {
               [self sendResult:result forUser:nil error:error];
             }];
+    [[FIRAuth auth].currentUser updatePassword:password
```

Comment:
Could we handle this in another PR. Removing this and only handling unlink in this PR. Thanks.

Reply:
I fixed it.
Could you review again?

Reply:
You may not know the change point with force push.
The changed point is two.
* I fixed it to handle only A.
* [FIRAuth auth] -> [self getAuth:call.arguments] in ```unlink```.

Reply:
@kroikie : I want to use this unlink function. Could you review this pull request as soon as possible?